src/test/java/com/puppycrawl/tools/checkstyle/grammar/AstRegressionTest.java

```diff
@@ -122,6 +122,12 @@ public void testTypecast() throws Exception {
                 getPath("InputRegressionJavaTypecast.java"));
     }
 
+    @Test
+    public void testCharLiteralSurrogatePair() throws Exception {
+        verifyAst(getPath("InputCharLiteralSurrogatePair.txt"),
```

Comment:
> antlr.TokenStreamRecognitionException: expecting ''', found '½' 

Reason for error is explained at https://github.com/checkstyle/checkstyle/pull/7886#discussion_r395356889 and is completely separate from changing Checker. We have UT specific code that is loading the file and not loading it as UTF8. It must be changed to ensure, just like Checker, we always load the files in UTF8 by default.

Reply:
Done.